datadog/dogstatsd/base.py

```diff
@@ -129,6 +129,9 @@ def decrement(self, metric, value=1, tags=None, sample_rate=1):
         >>> statsd.decrement('files.remaining')
         >>> statsd.decrement('active.connections', 2)
         """
+        if value is None:
+            return
```

Comment:
Can we replace this by:
```python

metric_value = -value if value else value
self._report(metric, 'c', metric_value, tags, sample_rate)
```

I like the idea of keeping the `if value is None:` assertion within the `_report` method.